package.json

```diff
@@ -3,6 +3,7 @@
   "version": "6.0.0",
   "description": "high performance css-in-js",
   "main": "lib/index.js",
+  "module": "src/",
```

Comment:
I took a shortcut here. I don't know how you want to handle the ESM build of Emotion.